deploy/deploy.sh

```diff
@@ -39,14 +39,25 @@ retry 20 5 oc get address/workload-app.queue-requests -n $NS -o 'jsonpath={.stat
 AMQ_ADDRESS="amqps://$(oc get addressspace/workload-app -n $NS -o 'jsonpath={.status.endpointStatuses[?(@.name=="messaging")].serviceHost}')"
 AMQ_QUEUE="/$(oc get address/workload-app.queue-requests -n $NS -o 'jsonpath={.spec.address}')"
 
+#SSO credentials
+RHSSO_SERVER_URL="https://$(oc get routes -n redhat-rhmi-rhsso keycloak-edge -o 'jsonpath={.spec.host}')"
```

Comment:
I thought we should test the user sso and not the cluster one `redhat-rhmi-user-sso`

Reply:
I cant remember what was decided here, or really ever talking about it tbh, is there any difference doing one over the other?

Reply:
Hmm, Just noticed now in the task description, it should be user RHSSO. I was sticking to the RHSSO in the task title 😀. I am also not sure about any significant difference in doing one over the other

Reply:
OK, if we said user rhsso, i guess we better change it here.

Reply:
The user sso is the instance used by the customer to provide sso access to them for their users to their apps, therefore from an availability point of view is more critical that the internal sso that is used only to login to the RHMI consoles, that's why I remember we said that we were going to test the user sso

Reply:
Good catch @b1zzu